Removing CHI_RESOURCES/Dependencies.

wdhawkins opened the pull request:
The packages that were located in CHI_RESOURCES/Dependencies now need to be installed outside of ChiTech with other dependencies (MPI, PETSc, BOOST, VTK).  Modifying configure.sh and installation instructions appropriately.  Both sets of install instructions still need a bit of work.

mhanus commented on CMakeLists.txt:
```diff
@@ -77,18 +98,17 @@ endif()
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-# Force -O3 in release builds (some OS's might downgrade it)
```
Like above: was removing this line intentional?

mhanus commented on CMakeLists.txt:
```diff
@@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.2)
 
-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")
```
Was there a reason to remove this line? I added it only recently, just to make it more transparent to the user what is being built.

mhanus commented on CMakeLists.txt:
```diff
@@ -1,7 +1,6 @@
 
-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")
-
+#set(CMAKE_VERBOSE_MAKEFILE ON)
```
Again, was there a reason to add this line? I removed it just recently, because now you can specify the additional CMake flags on the command-line (`configure.sh -DCMAKE_VERBOSE_MAKEFILE=ON`).

This and a few other changes here seem to me to revert CMakeLists.txt to an older version – I guess it's accidental?

mhanus commented on CMakeLists.txt:
```diff
@@ -107,7 +127,8 @@ find_package(VTK COMPONENTS
 	vtkIOXML vtkParallelCore
 	vtkIOParallelXML REQUIRED)
 if (NOT VTK_FOUND)
-  message(FATAL_ERROR "VTK not found: ${VTK_NOT_FOUND_MESSAGE}")
```
Like above: looks like unintentional revert to older version of CMakeLists.txt

wdhawkins commented:
Yes, this looks like an unintentional revert to an older version.  Strange since I made sure I pulled the latest changes.  I’ll fix it.

— Daryl

On Sep 19, 2019, at 12:53 AM, mhanus <notifications@github.com<mailto:notifications@github.com>> wrote:


@mhanus requested changes on this pull request.

________________________________

In CMakeLists.txt<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chi-2Dtech_chi-2Dtech_pull_16-23discussion-5Fr325997333&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=ZA-4-3GIIK-J0Nj0kkLRPXEVv22sphlnitCiG_wogmo&e=>:

> @@ -77,18 +98,17 @@ endif()
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_EXTENSIONS OFF)

-# Force -O3 in release builds (some OS's might downgrade it)


Like above: was removing this line intentional?

________________________________

In CMakeLists.txt<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chi-2Dtech_chi-2Dtech_pull_16-23discussion-5Fr325997506&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=xtfrq512SWemfuZC8j7gGcmbT22i7FaCWGqZFd2Ys0k&e=>:

> @@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.2)

-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")


Was there a reason to remove this line? I added it only recently, just to make it more transparent to the user what is being built.

________________________________

In CMakeLists.txt<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chi-2Dtech_chi-2Dtech_pull_16-23discussion-5Fr325998132&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=A7Zd42znqDVSD1MxR_9uAIL4OXAM00I6RSOeViaQLnU&e=>:

> @@ -1,7 +1,6 @@
 cmake_minimum_required(VERSION 3.2)

-message(STATUS "Configuring ${CMAKE_BUILD_TYPE} build of ChiTech.")
-
+#set(CMAKE_VERBOSE_MAKEFILE ON)


Again, was there a reason to add this line? I removed it just recently, because now you can specify the additional CMake flags on the command-line (configure.sh -DCMAKE_VERBOSE_MAKEFILE=ON).

This and a few other changes here seem to me to revert CMakeLists.txt to an older version – I guess it's accidental?

________________________________

In CMakeLists.txt<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chi-2Dtech_chi-2Dtech_pull_16-23discussion-5Fr325998366&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=SYlgAoJ2sVnkWN3-nq_L1zeyP6mc8k3pPyMTBfeVQB4&e=>:

> @@ -107,7 +127,8 @@ find_package(VTK COMPONENTS
        vtkIOXML vtkParallelCore
        vtkIOParallelXML REQUIRED)
 if (NOT VTK_FOUND)
-  message(FATAL_ERROR "VTK not found: ${VTK_NOT_FOUND_MESSAGE}")


Like above: looks like unintentional revert to older version of CMakeLists.txt

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_chi-2Dtech_chi-2Dtech_pull_16-3Femail-5Fsource-3Dnotifications-26email-5Ftoken-3DACPYQZZQIHFYEW3Z6F35MZTQKMHT3A5CNFSM4IYCXTF2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCFHCFPQ-23pullrequestreview-2D290333374&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=4mTieXxtapWgIN4r8fzDJZD8fofTNpiIrEFu8YLIuFA&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ACPYQZZ24TGRQR2FBSYMPJTQKMHT3ANCNFSM4IYCXTFQ&d=DwMFaQ&c=u6LDEWzohnDQ01ySGnxMzg&r=AGGDkeGBg9rN2A12Ap4nyZd7ljYaFtmxn4Ih6xnU23U&m=UXGC9663IX9ZuijTzycj9OqmgrNFW-P5_5RP4gKaxN8&s=rH-8PrDZDaLPsmhMqGX9Na-Peg8ilOKWStsk1h0TRH0&e=>.
